gfx.cpp

```diff
@@ -68,6 +68,17 @@ SOFTWARE.
 #    pragma warning(pop)
 #endif
 
+extern "C"
+{
+__declspec(dllexport) extern const UINT D3D12SDKVersion = 613;
+__declspec(dllexport) extern char8_t const *D3D12SDKPath = u8".\\";
+
+__declspec(dllexport) UINT GetD3D12SDKVersion()
+{
+    return D3D12SDKVersion;
+}
+}
```

Comment:
The agility SDK variables are exported directly from gfx, this way any project using gfx doesnt have to be updated to define them if they didnt already. It does mean that any project that already was pulling in the agility sdk will have to remove their equivalent of these as otherwise they would clash (and it doesnt make sense for a parent project to pull in agility anymore as that could create versioning issues etc.).
The GetD3D12SDKVersion function is there to force the compiler to export these symbols as gfx is built as a static lib (and thus doesnt have exports) so we need to force exporting of these symbols from whatever exe/dll imports gfx